frontend/app/src/components/settings/data-security/data-management/PurgeImagesCache.vue

```diff
@@ -79,62 +79,66 @@ const css = useCssModule();
       </div>
     </div>
 
-    <VRow class="mb-0" align="center">
-      <VCol cols="12" :md="true" class="mb-n7 mb-md-0">
-        <VAutocomplete
-          v-model="source"
-          outlined
-          :label="t('data_management.purge_images_cache.select_image_source')"
-          :items="purgable"
-          item-text="text"
-          item-value="id"
-          :disabled="pending"
-        />
-      </VCol>
-      <VCol md="6">
-        <AssetSelect
-          v-if="source === PurgeableImageCache.ASSET_ICONS"
-          v-model="assetToClear"
-          outlined
-          persistent-hint
-          :label="t('data_management.purge_images_cache.label.asset_to_clear')"
-          :hint="t('data_management.purge_images_cache.hint')"
-        />
-        <VCombobox
-          v-else
-          v-model="ensToClear"
-          :items="ensNamesList"
-          outlined
-          :class="css['ens-input']"
-          chips
-          deletable-chips
-          clearable
-          :label="t('data_management.purge_images_cache.label.ens_to_clear')"
-          :hint="t('data_management.purge_images_cache.hint')"
-          multiple
-          persistent-hint
-        />
-      </VCol>
+    <div class="flex items-center gap-4">
+      <div class="flex flex-col md:flex-row md:gap-4 flex-1">
+        <div class="flex-1">
+          <VAutocomplete
+            v-model="source"
+            outlined
+            :label="t('data_management.purge_images_cache.select_image_source')"
+            :items="purgable"
+            item-text="text"
+            item-value="id"
+            :disabled="pending"
+          />
+        </div>
+        <div class="flex-1">
+          <AssetSelect
+            v-if="source === PurgeableImageCache.ASSET_ICONS"
+            v-model="assetToClear"
+            outlined
+            persistent-hint
+            :label="
+              t('data_management.purge_images_cache.label.asset_to_clear')
+            "
+            :hint="t('data_management.purge_images_cache.hint')"
+          />
+          <VCombobox
+            v-else
+            v-model="ensToClear"
+            :items="ensNamesList"
+            outlined
+            :class="css['ens-input']"
+            chips
+            deletable-chips
+            clearable
+            :label="t('data_management.purge_images_cache.label.ens_to_clear')"
+            :hint="t('data_management.purge_images_cache.hint')"
+            multiple
+            persistent-hint
+          />
+        </div>
+      </div>
 
-      <VCol cols="auto">
-        <VTooltip open-delay="400" top>
+      <div class="-mt-8">
```

Comment:
also here might be possible to remove the extra div and put the class on the RuiTooltip